src/helpers/isHostElementPointerEventEnabled.ts

```diff
@@ -0,0 +1,20 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import { getHostParent } from './component-tree';
+
+export const isHostElementPointerEventEnabled = (
```

Comment:
It would be helpful if this method contained code comment about how `pointerEvents` exactly work, how different values affect the enabled status of an element and its descenants.